src/status_im/constants.cljs

```diff
@@ -287,4 +287,4 @@
 (def swarm-proto-code "e4")
 
 (def faq "https://status.im/faq/")
-(def faq-keycard (str faq "#keycard"))
+(def faq-keycard "https://status.im/keycard-integration")
```

Comment:
i just changed it here https://github.com/status-im/status-react/pull/10723/files, https://github.com/status-im/status-react/issues/10718 , and i feel like we need to keep #keycard in some cases @hesterbruikman

Reply:
Keycard onboarding > Learn more https://status.im/faq/#keycard
Settings > Keycard > Helphttps://status.im/faq/#keycard

Reply:
in onboarding the link should be https://status.im/keycard-integration and not https://status.im/faq/#keycard because at onboarding stage we are also trying to explain what is keycard, and push users to buy it too. 
https://status.im/keycard-integration is a general page which explains what is keycard, has a buy button, and can also a link to the FAQ

Reply:
Apologies @flexsurfer ! @guylouis is right. Keycard onboarding and pairing is updated in this PR to https://status.im/keycard-integration. I should have caught this 7 days ago. `Settings` > `Keycard`  remains https://status.im/faq/#keycard as you just merged in #10723

Reply:
thanks, so @rasom  probably we need to add new constant use faq constant for `Settings > Keycard`